lblod/frontend-lokaal-mandatenbeheer
# LMB-457:  Disable mandatarissen that are in completed legislatuur

## Description

When a legislatuur is status completed it cannot be edited anymore. In the legislatuur all mandatarissen can also not be edited anymore as they are final. When going to a specific mandataris in the period all the edit buttons will be disabled.

## How to test

1. Change status of legislatuur to "Behandeld"
2. Check out the mandatarissen in the periode of the legislatuur
3. They should not be editable anymore

## Attachments

From Preparation

![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/36266973/7bd78cd9-248f-4f46-9ede-65e17fa2d1b7)


To behandeld

![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/36266973/af2d48ab-3ad7-4ded-9160-9e71a6db642a)
